packages/backend/src/ee/services/AiAgentService.ts

```diff
@@ -2254,7 +2254,7 @@ export class AiAgentService {
             );
         }
 
-        const slackTagRegex = /^.*?(<@U\d+\w*?>)/g;
+        const slackTagRegex = /<@U\d+\w*?>/g;
```

Comment:
I think the better way to handle this is to replace slack id with actual name wether it's bot or user